manipulation_test.go

```diff
@@ -56,6 +56,13 @@ func TestAfterHtml(t *testing.T) {
 	printSel(t, doc.Selection)
 }
 
+func TestAfterHtmlContext(t *testing.T) {
```

Comment:
General comment for tests of methods that change the HTML, I think it's easier to read and setup the test by using a local string as data (instead of using the helper `DocXxx` functions), and parsing with `NewDocumentFromReader(strings.NewReader(data))`.